### DISCUSSION TITLE
Timeout customization

### DISCUSSION THREAD

**Discussion by hy0gy** [WangYihang/Platypus#79] [Ideas]
Would be really nice to be able to customize while executing Platypus all the relative timeouts expected once the shell from the victims arrive

**Comment by WangYihang** [comment]
Great idea, this will be implemented in the next release. Thanks for your advice.

**Reply by hy0gy** [reply]
Sorry if I write to you again, but I downloaded the last version (v1.5.1) and I did not see anything related to the timeouts in the options
How do I change them? Or I'am I too early?

**Reply by WangYihang** [reply]
I'm sorry I have been busy these days, your advice has not been processed yet.  : (

The v1.5.1 is just a pre-release. The only difference between v1.5.0 and v1.5.1 is: 

```
** BUG FIX **

1. fix /api/server crash #69 #105 #106 #107
```
